sdk/attestation/azure-security-attestation/azure/security/attestation/_client.py

```diff
@@ -22,107 +18,135 @@
+        if the token is invalid, the `validation_callback` function should throw
+        an exception.
+    :paramtype validation_callback: ~typing.Callable[[AttestationToken, AttestationSigner], None]
+    :keyword bool validate_signature: If True, validate the signature of the token being validated.
+    :keyword bool validate_expiration: If True, validate the expiration time of the token being validated.
+    :keyword str issuer: Expected issuer, used if `validate_issuer` is true.
+    :keyword float validation_slack: Slack time for validation - tolerance applied
+        to help account for clock drift between the issuer and the current machine.
+    :keyword bool validate_issuer: If True, validate that the issuer of the token matches the expected issuer.
+    :keyword bool validate_not_before_time: If true, validate the "Not Before" time in the token.
+
+    .. tip::
+        The `validate_token`, `validation_callback`, `validate_signature`,
+        `validate_expiration`, `validate_not_before_time`, `validate_issuer`, and
+        `issuer` keyword arguments are default values applied to each API call within
+        the :py:class:`AttestationClient` class. These values can be
+        overridden on individual API calls as needed.
+
+    For additional client creation configuration options, please see `Python Request
+    Options <https://aka.ms/azsdk/python/options>`_.
 
     """
 
-    def __init__(
-        self,
-        credential,  # type: "TokenCredential"
-        instance_url,  # type: str
-        **kwargs  # type: Any
-    ):
-        # type: (TokenCredential, str, Any) -> None
+    def __init__(self, endpoint, credential, **kwargs):
+        # type: (str, TokenCredential, Any) -> None
 
         if not credential:
             raise ValueError("Missing credential.")
-        self._config = AttestationClientConfiguration(credential, instance_url, **kwargs)
-        self._client = AzureAttestationRestClient(credential, instance_url, **kwargs)
+        self._config = AttestationClientConfiguration(**kwargs)
+        self._client = AzureAttestationRestClient(credential, endpoint, **kwargs)
         self._statelock = Lock()
         self._signing_certificates = None
 
     @distributed_trace
-    def get_openidmetadata(self, **kwargs):
-        #type:(Dict[str, Any]) -> Any
-        """ Retrieves the OpenID metadata configuration document for this attestation instance.
+    def get_open_id_metadata(self, **kwargs):
+        # type: (Dict[str, Any]) -> Dict[str, Any]
+        """Retrieves the OpenID metadata configuration document for this attestation instance.
+
+        The metadata configuration document is defined in the `OpenID Connect
+        Discovery <https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationResponse>`
+        specification.
+
+        The attestation service currently returns the following fields:
+        * issuer
```

Comment:
To render the link and bulleted list correctly:
```suggestion
        Discovery <https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationResponse>`_
        specification.

        The attestation service currently returns the following fields:

        * issuer
```